src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -3822,6 +3823,43 @@ void recallTlrShouldNotBeCreatedForInstanceWithOnlyAgedToLostItem() {
       "Request has no loan or recallable item")));
   }
 
+  @Test
+  void recallTlrShouldBeCreatedForOnOrderItemIfInstanceHasOnOrderAndDeclaredLostItems() {
+    configurationsFixture.enableTlrFeature();
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    UUID instanceId = UUID.randomUUID();
+
+    var item = buildItem(instanceId, "111");
+    var loan = checkOutFixture.checkOutByBarcode(item, usersFixture.charlotte());
+    IndividualResource firstRequest = requestsClient.create(buildRecallTitleLevelRequest(
+      usersFixture.steve().getId(), pickupServicePointId, instanceId));
+
+    assertThat(firstRequest.getResponse(), hasStatus(HTTP_CREATED));
+    JsonObject firstRequestJson = firstRequest.getJson();
+    assertThat(firstRequestJson.getString("requestLevel"), is(RequestLevel.TITLE.getValue()));
+    assertThat(firstRequestJson.getString("requestType"), is(RECALL.getValue()));
+    assertThat(firstRequestJson.getString("instanceId"), is(instanceId.toString()));
+    assertThat(firstRequestJson.getString("itemId"), is(item.getId()));
+    assertThat(firstRequestJson.getJsonObject("item").getString("status"), is("Checked out"));
+    declareLostFixtures.declareItemLost(loan.getJson());
+
```

Comment:
check that status changed to Declared lost

Reply:
done